### PR TITLE
Consider empty change as warning instead of error

### PR DESCRIPTION
**Pull request by pierretr**
This allows pushing changesets for multiple stacks even if some do not
get updated.

### CODE REVIEW

File: src/e3/aws/cfn/main.py [hunk 1]
```diff
@@ -184,9 +184,19 @@ def execute_for_stack(self, stack: Stack) -> int:
                         result = stack.describe_change_set(changeset_name)
 
                     if result["Status"] == "FAILED":
-                        logging.error(result["StatusReason"])
+                        change_failed = False
```
Comment (enzbang) [inline src/e3/aws/cfn/main.py:187]: I would prefer avoiding double negation, you can pick another variable name such as "success" or "change_executed"?

File: tests/tests_e3_aws/cfn/main/main_test.py [hunk 2]
```diff
@@ -67,6 +73,107 @@ def create_stack(self):
...
+)
+def test_cfn_main_push_existing_stack(
+    status: Tuple[str, str, int], monkeypatch
+) -> None:
+    """Test pushing an already existing stack.
+
+    :param status: Tuple with status and status reason from describe_change_set
+        response and associated expected execute return value.
+    """
+
```
Comment (enzbang) [inline tests/tests_e3_aws/cfn/main/main_test.py:99]: no empty line after docstring

**Review by enzbang — COMMENTED** [review]
LGTM: some minor comments

File: tests/tests_e3_aws/cfn/main/main_test.py (hunk 2 above)
Comment (pierretr) [reply tests/tests_e3_aws/cfn/main/main_test.py:99]: In this case black adds an empty line.

**Review by enzbang — APPROVED** [review]

**Pull request merged**